integreat_cms/cms/templates/push_notifications/push_notification_form.html

```diff
@@ -160,6 +160,13 @@ <h3 class="heading font-bold text-black">
                                 {{ push_notification_form.schedule_send.label }}
                             </label>
                         </div>
+                        {% if push_notification_form.regions|length == 1 %}
+                            <span>
+                                {% blocktranslate trimmed %}
+                                    This region reached an average of {{ avg_accepted_notifications }} devices last week.
+                                {% endblocktranslate %}
+                            </span>
+                        {% endif %}
```

Comment:
The issue is about region managers, so I decided to display the text only if the user has access to exactly one region. Does that approach make sense?

Reply:
I'm confused why this should be displayed exclusively to region managers and not to all roles region managers *and higher* (like staff), but I am new to the discussion around the original feature request so I will stop at making this regard only.

I think functionality about authentication / showing data only for certain roles should be done on the python side, there it's both easier to implement and able to be done so explicitly. I don't know if your assumption would hold up, but this should be more robust in any case.
For communicating that to the template I suggest setting `avg_accepted_notifications` to `None` if the user is not allowed to view that data, and then displaying it in the template based on that. If you want to handle the case that data might not be available (different from `None`), I guess using a separate boolean instead for saying to show it at all would be the next best option.